internal/host/bruteforce/modules/ssh.go

```diff
@@ -0,0 +1,62 @@
+package host
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/Method-Security/networkscan/generated/go/bruteforce"
+	"golang.org/x/crypto/ssh"
+)
+
+type SSHLibrary struct{}
+
+func (SSHLib *SSHLibrary) StandardPorts() []int {
+	return []int{22, 2222}
+}
+
+func (SSHLib *SSHLibrary) BruteForce(host string, port int, timeout int, options map[string]interface{}) (*bruteforce.AttemptInfo, []string) {
+	attempt := bruteforce.AttemptInfo{Timestamp: time.Now()}
+	errors := []string{}
+
+	username := options["username"].(string)
+	password := options["password"].(string)
+	request := bruteforce.SshRequestInfo{Username: username, Password: password, Host: host, Port: port}
+	response := bruteforce.SshResponseInfo{}
+
+	config := &ssh.ClientConfig{
+		User:            username,
+		Auth:            []ssh.AuthMethod{ssh.Password(password)},
```

Comment:
Have you thought about what we'll do if they require PKI? Not saying we should implement it yet, but is there an error we get back that would let us know they require a key

Reply:
I have not, for v1 ill do some digging on how to test this locally and try to find a pattern